c2cgeoform/routes.py

```diff
@@ -1,6 +1,4 @@
-
-
-tables=[]
```

Comment:
If tables is not global, you lose the possibility to call register_models many times